api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupMERQuarterly24.java

```diff
@@ -96,11 +96,13 @@ public ReportDefinition constructReportDefinition() {
     rd.setDescription(getDescription());
     rd.setParameters(txPvlsDataset.getParameters());
     rd.addDataSetDefinition("N", Mapped.mapStraightThrough(txNewDataset.constructTxNewDataset()));
-    rd.addDataSetDefinition(
-        "C", Mapped.mapStraightThrough(txCurrDataset.constructTxCurrDataset(true)));
+    /*rd.addDataSetDefinition(
```

Comment:
Don't comment the code. Instead delete this lines.

Reply:
Commented code removed.